app/content/react/guides/query-parameters.md

```diff
@@ -0,0 +1,124 @@
+## Query Parameters
+
+You can use QueryParameters to filter your layers & widgets at the backend side, using the concept of *parametrized queries* over your sources. You can check our related [deck.gl documentation]({{< ref "/deck-gl/reference#queryparameters-depends-on-provider-optional" >}} "About Us") for different providers.
+In this guide we're going to create a simple map that shows the earthquakes in Spain and allows to filter it by their magnitude.
+
+#### Creating a map using QueryParameters
+
+First you'll need to create a source that supports query parameters. In this case we are getting all the earthquakes whoses magnitude are between `min: 4` and `max: 5`.
+
+```ts
+import { MAP_TYPES } from '@deck.gl/carto';
+
+const sql = 'select * from `carto-demo-data.demo_tables.spain_earthquakes` where magnitude between @min and @max';
```

Comment:
This syntax (parametrized queries) is only compatible with BigQuery. I think that we should specify it. What about mentioning how would be the same query with another provider

Reply:
it's just an example. I've added a link to the documentation for queryParameters, where different providers are specified. Anyways I think that would be great to specify that this example is based on BQ

Reply:
I agree with @Josmorsot it'd be enough to point that this example is for BQ and then link to the `queryParameters` documentation with the specifics for each provider.